docs/source/tutorials/outliers.ipynb

```diff
@@ -318,35 +349,38 @@
   },
   {
    "cell_type": "markdown",
-   "id": "f31f7725",
+   "id": "ad857d69",
    "metadata": {},
    "source": [
-    "## 4. Use cleanlab to find outliers in the dataset\n",
+    "## 4. Use cleanlab and `feature_embeddings` to find outliers in the dataset\n",
     "\n",
     "#### Scoring outliers in training data\n",
     "\n",
-    "Calling cleanlab's ``get_outlier_score()`` on ``train_feature_embeddings`` will find any naturally occuring outliers in ``train_data``. These examples are atypical images that look strange or different from the majority of examples in the dataset. In our case, these correspond to odd-looking images of animals that do not resemble typical animals depicted in **cifar10**. This method produces a score in [0,1] for each example, where lower values correspond to more atypical examples (more likely out-of-distribution)."
+    "Fitting cleanlab's ``OutOfDistribution`` class on ``train_feature_embeddings`` will find any naturally occuring outliers in ``train_data``. These examples are atypical images that look strange or different from the majority of examples in the dataset. In our case, these correspond to odd-looking images of animals that do not resemble typical animals depicted in **cifar10**. This method produces a score in [0,1] for each example, where lower values correspond to more atypical examples (more likely out-of-distribution)."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "aef10ced",
+   "id": "feb0f519",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Get outlier scores for each of our train_data feature embeddings\n",
-    "train_outlier_scores, knn = get_outlier_scores(features=train_feature_embeddings, return_estimator=True)\n",
+    "# Create OutOfDistribution object\n",
```

Comment:
Such comments probably aren't necessary when the code is self-explanatory.  This notebook is on the longer side, so shortening it at any opportunity would be good!